extensions/resource-deployment/src/ui/deployClusterWizard/constants.ts

```diff
@@ -17,8 +17,10 @@ export const DomainDNSName_VariableName = 'AZDATA_NB_VAR_BDC_AD_DOMAIN_DNS_NAME'
 export const Realm_VariableName = 'AZDATA_NB_VAR_BDC_AD_REALM';
 export const DomainServiceAccountUserName_VariableName = 'AZDATA_NB_VAR_BDC_AD_DOMAIN_SVC_USERNAME';
 export const DomainServiceAccountPassword_VariableName = 'AZDATA_NB_VAR_BDC_AD_DOMAIN_SVC_PASSWORD';
-export const AppOwners_VariableName = 'AZDATA_NB_VAR_AD_BDC_APP_OWNERS';
-export const AppReaders_VariableName = 'AZDATA_NB_VAR_AD_BDC_APP_READERS';
+export const AppOwners_VariableName = 'AZDATA_NB_VAR_BDC_AD_APP_OWNERS';
+export const AppReaders_VariableName = 'AZDATA_NB_VAR_AD_BDC_AD_APP_READERS';
+export const SubDomain_VariableName = 'AZDATA_NB_VAR_BDC_AD_SUBDOMAIN';
```

Comment:
Shouldn't you be updating the required azdata version as well?

Reply:
i am following up separately for the version update. will have a separate PR for that.